lol.janjic.Splitcat.yml

```diff
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: lol.janjic.Splitcat
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: splitcat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+modules:
+  - name: Splitcat
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - |
+        mv splitcat /app/
+        chmod +x /app/splitcat/splitcat
+        mkdir -p /app/bin
+        ln -s /app/splitcat/splitcat /app/bin/splitcat
+
+        wget https://raw.githubusercontent.com/vogonwann/splitcat/d8f95eff7dbfaf283dbfaf93b64ebbd2302d7847/linux/flutter/assets/icons/256x256/icon.png -o $iconDir/$projectId.png
+
+        install -Dm644 packaging/linux/$FLATPAK_ID.desktop -t $FLATPAK_DEST/share/applications/
+        install -Dm644 packaging/linux/$FLATPAK_ID.appdata.xml -t $FLATPAK_DEST/share/metainfo
+    sources:
+      - type: archive
+        url: https://github.com/vogonwann/splitcat/releases/download/v0.1.5/splitcat-linux-x64.tar.gz
```

Comment:
you have to restrict to x86_64 only https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson

unless you have aarch64 packages

Reply:
I have aarch64 packages, but not in the latest version. Let me build it.

Reply:
added >> `https://github.com/vogonwann/splitcat/releases/download/v0.1.5/splitcat-linux-arm64.tar.gz`

Reply:
how should I add it. I'm completely new to this. Sorry... Just to add another source or I have to specify architecture per source?

Reply:
see the first codeblock as an example https://docs.flatpak.org/en/latest/module-sources.html

Reply:
should be done now